src/app/services/views/ServiceDetailView.vue

```diff
@@ -4,222 +4,60 @@
     name="service-detail-view"
     data-testid="service-detail-view"
   >
-    <AppView
-      :breadcrumbs="[
-        {
-          to: {
-            name: 'mesh-detail-view',
-            params: {
-              mesh: route.params.mesh,
-            },
-          },
-          text: route.params.mesh,
-        },
-        {
-          to: {
-            name: 'services-list-view',
-            params: {
-              mesh: route.params.mesh,
-            },
-          },
-          text: t('services.routes.item.breadcrumbs')
-        },
-      ]"
-    >
+    <AppView>
       <template #title>
-        <h1>
+        <h2>
           <RouteTitle
-            :title="t('services.routes.item.title', {name: route.params.service})"
+            :title="t('services.routes.item.navigation.service-detail-view')"
             :render="true"
           />
-        </h1>
+        </h2>
       </template>
 
-      <DataSource
-        v-slot="{ data, error }: ServiceInsightSource"
-        :src="`/meshes/${route.params.mesh}/service-insights/${route.params.service}`"
-      >
-        <LoadingBlock v-if="data === undefined" />
-
-        <ErrorBlock
-          v-else-if="error"
-          :error="error"
-        />
-
-        <TabsWidget
-          v-else
-          :tabs="getTabs(data)"
+      <template v-if="props.data.serviceType === 'external'">
+        <DataSource
+          v-slot="{ data: externalService, error: externalServiceError }: ExternalServiceSource"
+          :src="`/meshes/${route.params.mesh}/external-services/${route.params.service}`"
         >
-          <template #overview>
-            <DataSource
-              v-if="data.serviceType === 'external'"
-              v-slot="{ data: externalService, error: externalServiceError }: ExternalServiceSource"
-              :src="`/meshes/${route.params.mesh}/external-services/${route.params.service}`"
-            >
-              <LoadingBlock v-if="externalService === undefined" />
-
-              <ErrorBlock
-                v-else-if="externalServiceError"
-                :error="externalServiceError"
-              />
+          <LoadingBlock v-if="externalService === undefined" />
```

Comment:
What happens if externalService is undefined (i.e. still loading) and we get an error?

Reply:
Then this stays in a loading state. I’ve changed the order for this instance now.